functorch/test/test_aotdispatch.py

```diff
@@ -728,6 +657,500 @@ def assert_compiler(gm: torch.fx.GraphModule, _):
+    xfail('trapezoid', ''),  # Cannot call sizes() on tensor with symbolic sizes/strides
+    xfail('trapz', ''),  # Cannot call sizes() on tensor with symbolic sizes/strides
+    xfail('triangular_solve', ''),  # aten.triangular_solve.default - couldn't find symbolic meta function/de...
+    xfail('tril', ''),  # prims::arange() Expected a value of type 'number' for argument 'end' but instead fo...
+    xfail('triu', ''),  # prims::arange() Expected a value of type 'number' for argument 'end' but instead fo...
+    xfail('true_divide', ''),  # 'int' and 'torch._C.SymIntNode'
+    xfail('trunc', ''),  # aten.zeros_like.default - couldn't find symbolic meta function/decomposition
+    xfail('unbind', ''),  # tensor_split() received an invalid combination of arguments - got (FakeTensor, torch...
+    xfail('unflatten', ''),  # Cannot call sizes() on tensor with symbolic sizes/strides
+    xfail('unfold', ''),  # could not find kernel
+    xfail('var', ''),  # Cannot call numel() on tensor with symbolic sizes/strides
+    xfail('var_mean', ''),  # Cannot call numel() on tensor with symbolic sizes/strides
+    xfail('view_as_complex', ''),  # aten.view_as_complex.default - couldn't find symbolic meta function/deco...
+    xfail('view_as', ''),  # Cannot call sizes() on tensor with symbolic sizes/strides
+    xfail('view', ''),  # Cannot call sizes() on tensor with symbolic sizes/strides
+    xfail('vsplit', ''),  # Cannot call sizes() on tensor with symbolic sizes/strides
+    xfail('vstack', ''),  # Cannot call sizes() on tensor with symbolic sizes/strides
+    xfail('xlogy', ''),  # 'int' and 'torch._C.SymIntNode'
+    xfail('zero_', ''),  # aten.zero_.default - couldn't find symbolic meta function/decomposition
+}
+
+def _test_aot_autograd_helper(self, device, dtype, op):
+    if not op.supports_autograd:
+        self.skipTest("Op does not support autograd")
+
+    sample_inputs_itr = op.sample_inputs(device, dtype, requires_grad=True)
+    for sample_input in sample_inputs_itr:
+        t_args = [sample_input.input] + list(sample_input.args)
+        t_kwargs = sample_input.kwargs
+        flat_args, args_spec = pytree.tree_flatten((t_args, t_kwargs))
+        sentinel_val = -42
+        is_tensor_spec = [sentinel_val if isinstance(arg, torch.Tensor) else arg for arg in flat_args]
+        args = [arg for arg in flat_args if isinstance(arg, torch.Tensor)]
+
+        def f(args):
+            cur_flat_args = list(is_tensor_spec)
+            args = iter(args)
+            for idx, v in enumerate(cur_flat_args):
+                if v == sentinel_val:
+                    cur_flat_args[idx] = next(args)
+            c_args, c_kwargs = pytree.tree_unflatten(cur_flat_args, args_spec)
+            return op.op(*c_args, **c_kwargs)
+
+        def call_forwards_backwards(f):
+            out = wrapper_set_seed(f, args)
+            if isinstance(out, tuple):
+                sm = 0
+                for i in out:
+                    sm += i.sum()
```

Comment:
This will inplace right? Well I guess we support differentiating it so it's fine?